src/Forms/GridField/GridFieldDataColumns.php

```diff
@@ -228,16 +229,18 @@ public function getColumnMetadata($gridField, $column)
      * @param ViewableData $record
      * @param string $columnName
      * @return string|null - returns null if it could not found a value
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it.
      */
     protected function getValueFromRelation($record, $columnName)
```

Comment:
Turns out this method just isn't used at all. It used to be used in `getColumnContent()` but the work this method used to do is now offloaded to `GridField::getDataFieldValue()`.

Found this because it's the only usage of `XML_val()` in core.